origen_site_config.yml

```diff
@@ -2,11 +2,51 @@
 
 # If your company has an internal gem server enter it here
 #gem_server: http://gems.company.net:9292
+
 # When true Origen will configure and run Bundler in a consistent way for every
 # user (recommended)
 gem_manage_bundler: true
-# Where user's local gems will be installed if gem_manage_bundler is set to true.
-gem_install_dir: ~/.origen/gems
+
+# Define where a typical user's home directory will be, with a hidden directory for Origen.
+home_dir: ~/.origen
+
+# Define where the gems should be installed
+# By default, this will be at <home_dir>/gems but can be overridden here.
+#gem_install_dir: ~/.origen/gems
+
+# ORIGEN STARTUP OPTIONS
+# These options define how and where Origen should boot.
+# For these options: 'Startup' refers to Origen running externally from any application instance.
+# 'Startup app' refers to the actual Origen instance that is running during Origen 'startup'
+# 'User install' refers to an Origen startup running out of the user's 'user_install_dir' directory.
+# 'Global install' refers to an Origen startup running out of some other location in the event of
+#   'user_install_enable' set to false, but 'global_install_dir' being defined.
+# Likewise:
+#   'user startup (app)' refers to starting up Origen from the user's home directory.
+#   'user startup (app)' refers to starting up Origen from the global install directory.
+#   'universal startup (app)' refers to starting up Origen from the universal directory, that is, the location that 
+#     Origen is sourced from (or where 'which origen' points to).
+# NOTE: If 'user_install_enable' is true AND 'user_install_dir' is valid, but non-existing, Origen will
+#  create a 'startup app' at 'user_install_dir'. BUT, if 'global_install_dir' is defined but that path is
+#  not existing (or no 'startup app' exists there), Origen will raise an error.
+
+# By default, don't allow user install. This is more for power users to utilize for debug or who want absolute control
+# over their environment.
+user_install_enable: false
+
+# Default to the user's install dir being the .origen in their home directory but allow it to be overridden
+#user_install_dir: ~/.origen
+
+# Default 'global_install_dir' to nil, meaning no 'global_install_dir' is present. If neither this nor user_install_enable
```

Comment:
No issue with using the tooL_repo terminology, just to note that the comments in this file need updated to reflect the new names